chapters/remote/jinad.md

```diff
@@ -1,88 +1,75 @@
 
-In the command below, we use the tag `latest-daemon` which uses the latest release version of `jinad`. Of course, you can switch to other versions, and you can find all the available versions at [hub.docker.com](https://hub.docker.com/repository/docker/jinaai/jina/tags?page=1&ordering=last_updated&name=daemon). You can found more information about the versioning at [github.com/jina-ai/jina](https://github.com/jina-ai/jina/blob/master/RELEASE.md).
+We use the tag `latest-daemon` in the below command, which uses the latest release of `JinaD`. To use other versions, please refer to [versioning in Jina](https://github.com/jina-ai/jina/blob/master/RELEASE.md) and find all the available versions at [Docker Hub](https://hub.docker.com/repository/docker/jinaai/jina/tags?page=1&ordering=last_updated&name=daemon).
 
 ```bash
 docker pull jinaai/jina:latest-daemon
 ```
 
-### Using PyPi package
+##### PyPi package
 
-> Notes: As one part of the jina package, `jinad` follows the same [installation instructions of jina](https://docs.jina.ai/chapters/install/os/via-pip.html) and you only need to cherry pick `[daemon]`
+> Note: As one part of the jina package, `JinaD` follows the same [installation instructions of jina](https://docs.jina.ai/chapters/install/os/via-pip.html) and you only need to cherry pick `[daemon]`
 
 On Linux/Mac, simply run:
 
 ```bash
 pip install "jina[daemon]"
 ```
 
-### Install from the Master Branch
+##### Master branch
 
 If you want to keep track of the master branch of our development repository:
 
 ```bash
 pip install "git+https://github.com/jina-ai/jina.git#egg=jina[daemon]"
 ```
 
-### Install from Your Local Fork/Clone
+##### Local fork/clone
 
-If you are a developer and want to test your changes on-the-fly: 
+If you are a developer and want to test your changes on-the-fly:
 
 ```bash
 git clone https://github.com/jina-ai/jina
 cd jina && pip install -e ".[daemon]"
-``` 
-
-## Usage 
-
-### Prerequisites
-Run `jinad` on the remote machine. We assume the remote is in the intranet and its IP address is `12.34.56.78`. By default, `jinad` will use the port `8000` for receiving requests. Make sure `8000` port is publicly accessible.
-
-After having `jinad` running on the remote, you can open the browser and visit `http://3.16.166.3:8000/alive` to check whether `jinad` is properly set up. 
-If everything works well, you will see the following response.
-
-```json
-{"status_code":200,"jina_version":"0.9.12"}
 ```
 
-#### Using Docker Container
+## Usage
+
+##### Docker Image (Recommended)
 
 We start a Docker container under the `host` mode so that it will connect all the ports to the host machine. `-d` option is to keep the container running in the background
 
 ```bash
 docker run -d --network host jinaai/jina:latest-daemon
 ```
 
-#### Using Native Python
+##### Native Python
+
+> Note: This doesn't ship fluentd, hence log streaming wouldn't be possible.
```

Comment:
Using Native Python doesn't ship fluentd, hence log streaming is not possible.

Reply:
Fixed